docs/native-tokens/minting.md

```diff
@@ -285,9 +283,14 @@ $ txix="0"
 $ funds="1000000000"
 $ policyid=$(cat policy/policyID)
 ```
+Also, transactions only used to *calculate* fees still must have a fee set, even though this fee parameter is ignored. (The calculated fee will be included the second time the command is run: when the *real* transaction is built.)  So this variable must also be set:
+
+```bash
+$ fee="0"
```

Comment:
I would not set the fee to 0 even in the dummy transaction. Transaction costs are based on number of bytes and in the cbor of the final transaction, a 0 takes up fewer bytes than say 300000. I would set the value to 30000 when building the dummy transaction because it's likely more than what the transaction will cost and will take up the right amount of bytes in the dummy transaction. Then, when your final transaction says it's going to cost you 170000 lovelace, you can be happy.

Reply:
This answer comes from experience in building very large transactions with JorManager. Once your transaction becomes very large with lots of tokens and things, setting a 0 fee causes you to have the wrong answer and too low of a fee. It's hard to go wrong with the 300000.
https://bitbucket.org/muamw10/jormanager/src/c5d59c5164fae1fc44393c01f3062d32a13e0275/src/main/kotlin/com/swiftmako/jormanager/controllers/WalletController.kt#lines-266

Reply:
thanks @AndrewWestberg - please see https://github.com/cardano-foundation/developer-portal/pull/283/commits/f249d430cad673c6eea6cbdf652798ac12e945f2